boardgame-ui/src/components/timer/TimerList.vue

```diff
@@ -0,0 +1,257 @@
+<template>
+  <div>
+      <div class="columns is-centered">
+        <div class="column">
+          <router-link
```

Comment:
When we click outside the timer box, on same horizontal line, we are redirected to timer, is it on purpose?

Reply:
@FServais

Reply:
No it's a bug to be fixed. Idk how yet. Will work on it :)

Reply:
Fixed :)